src/chain/amplitude/decoders/events/balances/transfer.ts

```diff
@@ -0,0 +1,14 @@
+import { events } from '../../../types';
+import { UnknownVersionError } from '../../../../../utils';
+import { Event, ITransferEventPalletDecoder } from '../../../../../indexer';
+
+export class TransferEventPalletDecoder implements ITransferEventPalletDecoder {
+  decode(event: Event) {
+    const { transfer } = events.balances;
+    if (transfer.v1.is(event)) {
+      return transfer.v1.decode(event);
+    } else {
+      throw new UnknownVersionError(event);
+    }
+  }
+}
```

Comment:
Some files are 2 spaces/tab others are 4 spaces/tab. Would be nice to have them all be the same but it would be too time consuming to do it 1 by 1 so only change it if you can globally change it for all files.

Reply:
I've added npm script to `check` and ` fix` prettier issues globally